src/components/Report/PdfReports/DefaultReportFooter.tsx

```diff
@@ -0,0 +1,22 @@
+import { View, StyleSheet } from '@react-pdf/renderer';
+import { memo } from 'react';
+
+// Create styles (pdf docs can't be given rem for some reason)
+const styles = StyleSheet.create({
+  footer: {
+    maxWidth: '100%',
+    width: '100%',
+    marginBottom: '25px',
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+  }
+});
+
+const StrategyTableFooter = () => {
```

Comment:
Should this be called DefaultReportFooter instead?

Reply:
Ah, good catch! I'll rename that.